.github/CODEOWNERS

```diff
@@ -0,0 +1,11 @@
+# Uberon code owners (Guardians of the main branch)
+# GitHub documentation about CODEOWNERS file - https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
+
+*       @anitacaron
```

Comment:
This makes @anitacaron being systematically asked to review _any_ PR that touches _any_ file!  (except the PRs that only touch YAML files or the Makefiles)

If she’s fine with that then I have no objection, but I am not sure anyone should be tasked with so much.

Reply:
That's fine.